code/src/WijmoProvider/Features/Selection.ts

```diff
@@ -146,10 +146,11 @@ namespace WijmoProvider.Feature {
         }
 
         public equalizeSelection(): OSFramework.OSStructure.CellRange[] {
-            //This method just makes sense for MultiRange
+            //This method just makes sense for MultiRange and for grid's without checked rows
             if (
                 this._grid.provider.selectionMode !==
-                wijmo.grid.SelectionMode.MultiRange
+                    wijmo.grid.SelectionMode.MultiRange ||
```

Comment:
just to make the point that you wrote "and for grid's without checked rows" and then used a || here

Reply:
changed it to "or"